### PR TITLE
Include parse to Bug field in Proposal

### LINKED ISSUE #3: Parse the "Bug" field from proposals
We currently don't parse the "bug" field from proposals

### PR DESCRIPTION
**Pull request by digoreis**
- Include field Bug in Proposal; closes #3

### CODE REVIEW

**Review by unnamedd — APPROVED** [review]

File: Sources/Bug.swift [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+//  License
+//  Copyright © 2016 Jesse Squires
+//  Released under an MIT license: http://opensource.org/licenses/MIT
+//
+
+import Foundation
+
+/// Represents a review manager for a proposal.
+public struct Bug {
+    public let code: String
```
Comment (jessesquires) [inline Sources/Bug.swift]: is this the `SR-0000` number? what about renaming this `srNumber` ?

File: Sources/Bug.swift [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+//
+//
+//  License
+//  Copyright © 2016 Jesse Squires
+//  Released under an MIT license: http://opensource.org/licenses/MIT
+//
+
+import Foundation
+
+/// Represents a review manager for a proposal.
```
Comment (jessesquires) [inline Sources/Bug.swift]: 🤔

**Review by jessesquires — CHANGES_REQUESTED** [review]
Thanks @digoreis !

A couple small things to address

File: Sources/Bug.swift (hunk 2 above)
Comment (digoreis) [reply Sources/Bug.swift]: 😞

**Comment by jessesquires** [comment]
Thanks @digoreis !

**Pull request merged**